src/nplinker/metabolomics/load_gnps.py

```diff
@@ -788,52 +427,14 @@ def load_dataset(strains,
     # if we don't have a quantification table, must be older-style dataset (or unknown format)
     if gnps_format != GNPS_FORMAT_NEW_FBMN and quant_table_file is None:
         logger.info('Found older-style GNPS dataset, no quantification table')
-        unknown_strains = load_clusterinfo_old(gnps_format, strains,
-                                               nodes_file, spec_dict)
+        unknown_strains = _load_clusterinfo_old(gnps_format, strains,
+                                                nodes_file, spec_dict)
     else:
         logger.info('quantification table exists, new-style GNPS dataset')
-        _, unknown_strains = load_clusterinfo_fbmn(strains, nodes_file,
-                                                   quant_table_file,
-                                                   metadata_table_file,
-                                                   spec_dict,
-                                                   ext_metadata_parsing)
-
-    molfams = make_families(spectra)
-    return spec_dict, spectra, molfams, unknown_strains
-
-
-def make_families(spectra):
-    families = []
-    family_dict = {}
-    family_index = 0
-    fams, singles = 0, 0
-    for spectrum in spectra:
-        family_id = spectrum.family_id
-        if family_id == -1:  # singleton
-            new_family = SingletonFamily()
-            new_family.id = family_index
-            family_index += 1
-
-            new_family.add_spectrum(spectrum)
-            spectrum.family = new_family
-            families.append(new_family)
-            singles += 1
-        else:
-            if family_id not in family_dict:
-                new_family = MolecularFamily(family_id)
-                new_family.id = family_index
-                new_family.family_id = family_id  # preserve original ID here
-                family_index += 1
-
-                new_family.add_spectrum(spectrum)
-                spectrum.family = new_family
-                families.append(new_family)
-                family_dict[family_id] = new_family
-                fams += 1
-            else:
-                family_dict[family_id].add_spectrum(spectrum)
-                spectrum.family = family_dict[family_id]
+        _, unknown_strains = _load_clusterinfo_fbmn(strains, nodes_file,
+                                                    quant_table_file,
+                                                    metadata_table_file,
+                                                    spec_dict,
+                                                    ext_metadata_parsing)
 
-    logger.debug('make_families: {} molams + {} singletons'.format(
-        fams, singles))
-    return families
+    return unknown_strains
```

Comment:
the return value is different with that of old function, why changing it? 
Did you check the code that calls this function?